liquibase-core/src/main/java/liquibase/change/core/CreateTableChange.java

```diff
@@ -142,7 +142,7 @@ public ValidationErrors validate(Database database) {
             String columnRemarks = StringUtil.trimToNull(column.getRemarks());
             if (columnRemarks != null) {
                 SetColumnRemarksStatement remarksStatement = new SetColumnRemarksStatement(catalogName, schemaName, tableName, column.getName(), columnRemarks, column.getType());
-                if (!(database instanceof MySQLDatabase) && SqlGeneratorFactory.getInstance().supports(remarksStatement, database)) {
+                if (SqlGeneratorFactory.getInstance().supports(remarksStatement, database)) {
```

Comment:
Any idea why this check about MySQL existed?

Reply:
Cannot say for comment on commit doesn't really say much but I tested both on MariaDB and MySql and it works fine